exams/views.py

```diff
@@ -1,12 +1,64 @@
 """
 Views for exams app
 """
+from datetime import datetime
+
+import pytz
 from django.views.generic.base import RedirectView
+from rest_framework import (
+    authentication,
+    permissions,
+    status,
+)
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from exams.api import sso_digest
+from exams.models import ExamProfile
 
 
 class PearsonCallbackRedirectView(RedirectView):
     """
     Redirect from Pearson callbacks to dashboard
     """
-    def get_redirect_url(self, status):  # pylint: disable=arguments-differ
-        return "/dashboard?exam={status}".format(status=status)
+    def get_redirect_url(self, status_code):  # pylint: disable=arguments-differ
+        return "/dashboard?exam={status_code}".format(status_code=status_code)
```

Comment:
I know this is technically unnecessary for this case, but can we use `quote_plus` for the query parameter?